champselect.py

```
@@ -0,0 +1,524 @@
+    }
+    response = connection.api_post("runes", request_body)
+    if response.status_code == 200:
+        # Success - the runepage was created successfully. Now return its data
+        u.print_and_write(f"Success! Created a rune page with id {response.json()["id"]}")
+        return response.json(), True
+
+    # No empty rune page slots
+    elif response.status_code == 400:
+        response_msg: str = response.json()["message"]
+        if response_msg != "Max pages reached":
+            raise RuntimeError(f"An error occured while trying to create a rune page: {response_msg}")
```

Ruff (0.13.1)

Avoid specifying long messages outside the exception class

(TRY003)